src/dmn/trn_rpc_protocol_handlers_1.c

```diff
@@ -1313,3 +1313,55 @@ int *update_transit_network_policy_1_svc(rpc_trn_vsip_cidr_t *policy, struct svc
 error:
 	return &result;
 }
+
+int *delete_transit_network_policy_1_svc(rpc_trn_vsip_cidr_key_t *policy_key, struct svc_req *rqstp)
+{
+	UNUSED(rqstp);
+	static int result;
+	int rc;
+	char *itf = policy_key->interface;
+	struct vsip_cidr_t cidr;
+
+	TRN_LOG_INFO("delete_transit_network_policy_1_svc service");
+
+	struct user_metadata_t *md = trn_itf_table_find(itf);
+	if (!md) {
+		TRN_LOG_ERROR("Cannot find interface metadata for %s", itf);
+		result = RPC_TRN_ERROR;
+		goto error;
+	}
+
+	cidr.tunnel_id = policy_key->tunid;
+	// Add explaination here for magic number 96
```

Comment:
consider adding following explanation:
cidr-related maps have tunnel-id(64 bits), local-ip(32 bits) prior to destination cidr; hence the final prefix length is 64+32+{cidr prefix}